tests/cli/commands/test_dag_command.py

```diff
@@ -47,6 +47,7 @@
 )
 
 
+# TODO: Check if tests needs side effects - locally there's missing DAG
```

Comment:
Can this be removed, looks like tests are passing

Reply:
See https://github.com/apache/airflow/pull/12704#discussion_r533789920